File: JavaBasic/139_count_consonants.java
// 139	139_count_consonants.java	Count the number of consonants in a given string using a loop.
public class _139_count_consonants {
    public static void main(String[] args) {
        String str = "Hello World";
        int count = 0;
        String vowels = "aeiouAEIOU";
        for (char c : str.toCharArray())
            if (Character.isLetter(c) && vowels.indexOf(c) == -1) count++;
        System.out.println("Consonants: " + count);
    }
}

File: JSBASIC/139_count_consonants.js
// 139	139_count_consonants.js	Count the number of consonants in a given string using a loop.

let str = "Hello World";
let count = 0;
let vowels = "aeiouAEIOU";
for (let i = 0; i < str.length; i++) {
    let ch = str[i];
    if (ch.match(/[a-zA-Z]/) && !vowels.includes(ch)) count++;
}
console.log("String:", str);
console.log("Number of consonants:", count);
